Expired/Views/CategoriesView.swift

```diff
@@ -31,6 +39,23 @@ struct CategoriesView: View {
                         .padding()
                 }
             })
+            .alert("Are you sure you want to delete this category?", isPresented: $showingDeleteAlert) {
+                Button("Maybe Later", role: .cancel) {
+                    deleteIndexSet = nil
+                }
+                Button("Yes", role: .destructive) {
+                    if let indexSet = deleteIndexSet {
+                        deleteCategory(indexSet: indexSet)
+                    }
+                    deleteIndexSet = nil
+                }
+            }
+            .alert("Unable To Delete", isPresented: $showingNotDeleteAlert , actions: {
+                Button("Ok", role: .cancel) {
```

Comment:
```suggestion
                Button("OK", role: .cancel) {
```

- uppercase the "OK" text